coop_env.py

```diff
@@ -58,8 +60,16 @@ def _kill_collided_cars(self):
             # Skip dead cars.
             if not(self._cars[i].is_alive):
                 continue
-            # Check if car i collided with any car.
             collided = False
+            # Check if car i collided with an obstacle
```

Comment:
Oh whoops looks like my car check is buggy. We should compute a list of cars to kill in the loop, and kill them outside the loop. This is because collisions are not transitive.